CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Update previous release bwc version to 2.4.0 ([#4455](https://github.com/opensearch-project/OpenSearch/pull/4455))
 - 2.3.0 release notes ([#4457](https://github.com/opensearch-project/OpenSearch/pull/4457))
 - Warn future developers not to pass headers to extensions ([#4430](https://github.com/opensearch-project/OpenSearch/pull/4430))
+- Principal Identity for Opensearch requests to/from extensions ([#4299](https://github.com/opensearch-project/OpenSearch/pull/4299))
```

Comment:
FYI, see #4477 and #4479.  This may be moved to a separate section in the future. Your thoughts on that issue and PR welcome!

Reply:
Should we have a section that keeps changelog in Extensions domain, so it doesn't mix up with changes in core.

Reply:
Yeah now that #4477 is merged you should add to that section.  Won't break anything but will add extra work when main releases new version or when we finally merge.